client/src/components/addSourceWizard/addSourceWizard.js

```diff
@@ -92,6 +94,33 @@ class AddSourceWizard extends React.Component {
     }
   }
 
+  setupSourceConnectionScan(source) {
+    const { addScan, startScan, deleteScan } = this.props;
+    const connectionId = _.get(source, 'connection.id');
+
+    const addUpdateScan = () => {
+      const srcId = _.get(source, 'id');
+      const srcName = _.get(source, 'name');
+
+      if (typeof srcId === 'number') {
+        let data = {
+          name: `Connection Scan ${srcName}`,
+          sources: [srcId]
+        };
+
+        return addScan(data).then(response => startScan(_.get(response, 'value.data.id')));
```

Comment:
Do we need to inform the user if either of these calls fail?

Reply:
We might, was looking at that... we could throw a similar error as below

Reply:
Added some additional catches for console warnings around the connection scans failing. For the most part though the app should probably continue unhindered.